pkg/config/validation.go

```diff
@@ -54,6 +55,23 @@ func validate(c *configapi.Configuration) field.ErrorList {
 	// Validate PodNamespaceSelector for the pod framework
 	allErrs = append(allErrs, validateIntegrations(c)...)
 
+	allErrs = append(allErrs, validateMultiKueue(c)...)
+
+	return allErrs
+}
+
+func validateMultiKueue(c *configapi.Configuration) field.ErrorList {
```

Comment:
I think we should also add validation for the `origin` label, using `IsValidLabelValue`.

Reply:
Great catch! Thank!

Reply:
Done.